<?xml version="1.0"?>
<launch>
    <arg name="team_side" default="home" />
    <arg name="sim_mode" default="true" />

    <group ns="game">
        <node name="game" pkg="aimbot" type="game_node.py" output="screen"/>
    </group>

</launch>